service/app/controllers/usermanagement/UserController.java

```diff
@@ -270,14 +269,13 @@
       String searchKey, String searchValue, Http.Request httpRequest) {
     HashMap<String, Object> map = new HashMap<>();
     String captcha = httpRequest.getQueryString(JsonKey.CAPTCHA_RESPONSE);
+    String mobileApp = httpRequest.getQueryString(JsonKey.MOBILE_APP);
     if (Boolean.parseBoolean(ProjectUtil.getConfigValue(JsonKey.ENABLE_CAPTCHA))
-        && StringUtils.isNotEmpty(captcha)) {
-      if (!CaptchaHelper.validate(captcha)) {
-        throw new ProjectCommonException(
-            ResponseCode.invalidCaptcha.getErrorCode(),
-            ResponseCode.invalidCaptcha.getErrorMessage(),
-            ResponseCode.IM_A_TEAPOT.getResponseCode());
-      }
+        && CaptchaHelper.validate(captcha, mobileApp)) {
```

Comment:
not comforting to note this inversion of CaptchaHelper.validate(). Pls ack if this is intended.